### PR TITLE
docs: redirect and broken link check

### PR DESCRIPTION
**Pull request by WeaponXII**
This PR resolves any malformed or broken links our manual audit found.

### CODE REVIEW

File: content/docs/conceptual-guides/branches.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Branches
+redirectFrom:
+  - /docs/cloud/concepts/
```
Comment (stepashka) [inline content/docs/conceptual-guides/branches.md:4]: not sure about this.
why do we want to redirect from cloud concepts here?

File: content/docs/cloud/concepts.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ A Neon user is identified by their email address.
 
 A user registers and authenticates in Neon Web UI with their GitHub or Google account. More authentication methods are coming soon.
 
-Once authenticated, a user can create and access [Projects](#project) and [query Project data](../tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
+Once authenticated, a user can create and access [Projects](#project) and [query Project data](../get-started-with-neon/tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
```
Comment (stepashka) [inline content/docs/cloud/concepts.md:12]: is this page available to users from the sidebar?
i can access it [directly](https://websitemain62807-redirects.gtsb.io/docs/cloud/concepts/), but i don't see how a user can discover it.

File: content/docs/get-started-with-neon/using-api-keys.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ A Neon user is identified by their email address.
 
 A user registers and authenticates in Neon Web UI with their GitHub or Google account. More authentication methods are coming soon.
 
-Once authenticated, a user can create and access [Projects](#project) and [query Project data](../tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
```
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:11]: projects and users were supposed to link to the sections that are currenly in this doc:
https://websitemain62807-redirects.gtsb.io/docs/cloud/concepts/

File: content/docs/cloud/concepts.md (hunk 2 above)
Comment (WeaponXII) [reply content/docs/cloud/concepts.md:12]: This page is now deprecated, each section was moved to a different page and either expanded on or rewritten.

File: content/docs/get-started-with-neon/using-api-keys.md (hunk 3 above)
Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md:11]: Projects doesn't seem to have been rewritten or moved yet. If the information is still up to date, I can move the project link (and the Postgres users) to the glossary and add those links accordingly.

File: content/docs/cloud/concepts.md (hunk 2 above)
Comment (stepashka) [reply content/docs/cloud/concepts.md:12]: > This page is now deprecated

Sorry, I didn't realize this.

> each section was moved to a different page

I couldn't find where the following sections were moved to: Neon User, Project, Postgres Users, Postgres Databases

Also the 'Compute Config' section does not belog to 'Technical Preview Free Tier'

Sorry I haven't noticed this earlier.

File: content/docs/get-started-with-neon/using-api-keys.md (hunk 3 above)
Comment (stepashka) [reply content/docs/get-started-with-neon/using-api-keys.md:11]: Projects look up to date.

Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md:11]: I've added Project to the glossary

File: content/docs/conceptual-guides/branches.md (hunk 1 above)
Comment (stepashka) [reply content/docs/conceptual-guides/branches.md:4]: Notes from the conversation today:
Add Branching to the Glossary (without removing the conceptual guides) and then move this redirection (of `/docs/cloud/concepts/`) to Glossary page.

File: content/docs/cloud/concepts.md (hunk 2 above)
Comment (stepashka) [reply content/docs/cloud/concepts.md:12]: Let's move the text of the Compute Config section to the Compute Lyfecycle page.

Comment (Harleta) [reply content/docs/cloud/concepts.md:12]: Hi @stepashka just updating you that Jabali update the link and move the branches to compute config. Can we merge this?

**Review by stepashka — APPROVED** [review]

**Comment by pkafei** [comment]
@stepashka We're waiting for @Harleta to review, and then we'll merge this PR into main.

File: content/docs/conceptual-guides/branches.md (hunk 1 above)
Comment (Harleta) [reply content/docs/conceptual-guides/branches.md]: Hi @stepashka thank you, we will make sure to do this.

**Comment by stepashka** [comment]
@Harleta ,@pkafei mentioned tha this is waiting for your review, did you have a chance to review this?

**Review by Harleta — APPROVED** [review]
Hi All, thank you for your patience. Did a final check on this PR and everything looks good to go!

**Pull request merged**